static/js/components/dashboard/courses/StatusMessages.js

```diff
@@ -113,6 +113,11 @@ const messageForAttemptedEdxExams = (course: Course, passedExam: boolean) => {
         </a>
       </span>
     )
+  } else if (!R.isEmpty(course.exams_schedulable_in_future)) {
+    return (
+      `${passedMsg} You can take the exam starting ` +
+      `on ${formatDate(course.exams_schedulable_in_future[0])}.`
```

Comment:
If this is the date _schedulable_ then the message should be a little different. 

```suggestion
      `${passedMsg} You can register to take the exam starting ` +
      `on ${formatDate(course.exams_schedulable_in_future[0])}.`
```